lib/ui.js

```diff
@@ -31,7 +31,13 @@ function prettyVersionDiff(oldVersion, inc) {
 module.exports = options => {
 	const pkg = util.readPkg();
 	const oldVersion = pkg.version;
-	const repositoryUrl = githubUrlFromGit(pkg.repository.url);
+	let repositoryUrl;
+	try {
+		repositoryUrl = githubUrlFromGit(pkg.repository.url);
+	} catch (err) {
+		const message = `${chalk.bold.magenta('Repository url')} not found in ${chalk.yellow('package.json')}`;
```

Comment:
Just set the message directly in the error.

Reply:
Done :)